osf/models/provider.py

```diff
@@ -240,8 +258,7 @@ def create_primary_collection_for_provider(sender, instance, created, **kwargs):
             instance.save()
         else:
             # A user is required for Collections / Groups
-            sentry.log_message('Unable to create primary_collection for CollectionProvider {}'.format(instance.name))
-
+            sentry.log_message('Unable to create primary_collection for CollectionProvider or RegistrationProvider {}'.format(instance.name))
```

Comment:
Minor: `instance.readable_type` could be used here to clarify which type it occurred for